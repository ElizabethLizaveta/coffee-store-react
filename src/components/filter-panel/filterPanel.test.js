import React from 'react'; 
import FilterPanel from './filterPanel';
import renderer from 'react-test-renderer'; 

describe('Testing <FilterPanel/>', () => {
it('FilterPanel have rendered correctly', () => {
    const char = renderer.create(<FilterPanel/>).toJSON();
    expect(char).toMatchSnapshot();
}); 
});