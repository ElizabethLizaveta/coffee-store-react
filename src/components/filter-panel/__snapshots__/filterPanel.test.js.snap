// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <FilterPanel/> FilterPanel have rendered correctly 1`] = `
<div
  className="shop__filter"
>
  <div
    className="shop__filter-label"
  >
    Or filter
  </div>
  <div
    className="shop__filter-group"
  >
    <button
      className="shop__filter-btn"
      onClick={[Function]}
      type="button"
    >
      Brazil
    </button>
    <button
      className="shop__filter-btn"
      onClick={[Function]}
      type="button"
    >
      Kenya
    </button>
    <button
      className="shop__filter-btn"
      onClick={[Function]}
      type="button"
    >
      Columbia
    </button>
  </div>
</div>
`;
